lib/hoost-edit.js

```diff
@@ -32,6 +32,7 @@ function editHost( ip, host, newip, newhost ) {
 
 function readFile( err, data ) {
   if( err ) throw err;
+  fs.chmodSync( file, '0644' );
```

Comment:
I think this is a bad practice. For changing a file, the user needs to have permission to do this.
To editing, use `sudo`.